platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/NetworkUtils.java

```diff
@@ -89,7 +90,9 @@ public static void handleNetworkException(final Exception e, final Connection co
      * @return the marker to use for logging
      */
     public static Marker determineExceptionMarker(final Exception e) {
-        return Utilities.isCausedByIOException(e) || Utilities.isRootCauseSuppliedType(e, SyncTimeoutException.class)
+        return Utilities.isCausedByIOException(e)
+                        || Utilities.isRootCauseSuppliedType(e, SyncTimeoutException.class)
+                        || Utilities.hasAnyCauseSuppliedType(e, SSLException.class)
```

Comment:
add a comment why we added this, with a link to the ticket

Reply:
added